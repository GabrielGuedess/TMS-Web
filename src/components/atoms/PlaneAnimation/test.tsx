import { render } from '@testing-library/react';

import { PlaneAnimation } from '.';

jest.mock('lottie-react', () => ({
  __esModule: true,
  default: function Mock() {
    return <div data-testid="Mock Lottie" />;
  },
}));

describe('<PlaneAnimation />', () => {
  it('should render the plane animation correctly', () => {
    const { container } = render(<PlaneAnimation />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
