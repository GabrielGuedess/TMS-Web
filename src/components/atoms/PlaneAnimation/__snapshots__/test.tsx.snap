// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaneAnimation /> should render the plane animation correctly 1`] = `
<div
  data-testid="Mock Lottie"
/>
`;
